tools/scuffed-scraper/main.py

```diff
@@ -0,0 +1,612 @@
+	"13 Department of Transport": None,  ## This Department Has no Courses
+	"22 Department of Health Technology": DTUDepartment.DTU_HEALTH_TECH,
+	"23 National Food Institute": DTUDepartment.DTU_FOOD,
+	"24 National Veterinary Institute": None,  ## This Department Has no Courses
+	"25 National Institute of Aquatic Resources": DTUDepartment.DTU_AQUA,
+	"26 Department of Chemistry": DTUDepartment.DTU_CHEMISTRY,
+	"27 Department of Biotechnology and Biomedicine": DTUDepartment.DTU_BIOENGINEERING,
+	"28 Department of Chemical Engineering": DTUDepartment.DTU_CHEMICAL_ENGINEERING,  # No exact match but close
+	"29 DTU Biosustain": DTUDepartment.DTU_BIOSUSTAIN,
+	"30 National Space Institute": DTUDepartment.DTU_SPACE,
+	"33 Department of Micro and Nanotechnology": None,  ## This Department Has no Courses
+	"34 Department of Electrical and Photonics Engineering": DTUDepartment.DTU_ELECTRO,
+	"36 DTU Bioinformatics": None,  ## This Department Has no Courses
+	"38 DTU Entrepreneurship": DTUDepartment.DTU_ENTREPRENEURSHIP,
+	"41 Department of Civil and Mechanical Engineering": DTUDepartment.DTU_CONSTRUCT,  # No exact match but close
+	"42 Department of Technology, Management and Economics": DTUDepartment.DTU_MANAGEMENT,
+	"46 Department of Wind and Energy Systems": DTUDepartment.DTU_WIND,
+	"47 Department of Energy Conversion and Storage": DTUDepartment.DTU_ENERGY,
+	"56 DTU Nanolab": DTUDepartment.DTU_NANOLAB,  # NOTE: Mismatch HTML ("59") value and label ("56").
+	"62 Department of Engineering Technology and Didactics": DTUDepartment.DTU_ENGINEERING_TECHNOLOGY,  # No exact match but close
+	"88 Other courses": None,  ## Either use "Department Involved", "External Institution", or use the Department of Course Responsible.
+}
+
+SCRAPE_CONTENT_TO_CONTENT = {
+	"General course objectives": "summary",
+	"Learning objectives": "objectives",
+	"Content": "content",
+	"CourseLiterature": "literature",
+	"Last updated": "updated",
+	"Remarks": "remarks",
+}
+
+SECTION_I18N = {
+	"summary": True,
+	"objectives": True,
+	"content": True,
+	"literature": False,
+	"updated": False,
+	"remarks": True,
+}
+
+
+####################
+# - Download Course
+####################
+class LangSessionCode(enum.StrEnum):
+	EN = "en-GB"
+	DK = "da-DK"
+
+
+LANG_TO_LANGSESSIONCODE = {
+	Lang.ENGLISH: LangSessionCode.EN,
+	Lang.DANISH: LangSessionCode.DK,
+}
+
+
+def g_session_cookies(
+	lang: LangSessionCode = LangSessionCode.EN,
+) -> requests.cookies.RequestsCookieJar:
+	cookie_jar = requests.get("https://kurser.dtu.dk").cookies
+	cookie_jar["{DTUCoursesPublicLanguage}"] = lang
+
+	return cookie_jar
+
+
+def g_course_soup(course_id: str, lang: Lang) -> BeautifulSoup:
+	path_scrape_cache = COURSE_SCRAPES_PATHS[lang] / f"{course_id}.html"
+	if path_scrape_cache.is_file():
+		with path_scrape_cache.open() as f:
+			return BeautifulSoup(f.read(), features="html.parser")
+
+	else:
+		print(
+			f"[https://kurser.dtu.dk/course/{course_id}] GET {lang} to {path_scrape_cache}...",
```

Comment:
is it black that adds the `,`? it hurts my eyes!! :face_with_head_bandage: ;)

Reply:
Not as such, but `black` allows it - it's `ruff` that's the culprit aha.

The nice thing about being so pedantic about `,` on all lines like this is cleaner `diff`s. If I were to want to print another line, the diff would only add a line; not add a line _and_ modify the existing line to add a `,`. In this case, sure, who cares - but being 100% consistent about it in general can have a good impact.

Reply:
I would prefer if you remove them. Thanks